package.json

```
@@ -14,7 +14,7 @@
     "fix": "yarn fix:sol && yarn fix:js",
     "fix:ts": "prettier --config .prettierrc.json './**/*.ts' --write",
     "fix:sol": "prettier --write 'contracts/**/*.sol'",
-    "size": "yarn run hardhat size-contracts",
+    "size": "npx hardhat size-contracts",
```

why not using forge ?
```suggestion
    "size": "forge build --sizes",
```

---

I shouldn't have changed this since it's out of the scope of this PR
I think one could argue that we shouldn't mix `hardhat` running on `nodejs` with `forge` which requires a separate installation, but I also think that `hardhat` should disappear progressively from this repository

---

Yes, poke #54